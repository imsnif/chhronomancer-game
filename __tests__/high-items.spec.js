import 'react-native'
import React from 'react'
import HighItems from '../components/high-items'
import renderer from 'react-test-renderer'

test('HighItems (default props) => renders correctly', () => {
  const tree = renderer.create(
    <HighItems />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('HighItems (with black item) => renders correctly', () => {
  const tree = renderer.create(
    <HighItems items={{black: true}} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
