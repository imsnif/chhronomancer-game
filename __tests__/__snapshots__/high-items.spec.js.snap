// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighItems (default props) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "black",
        "borderRadius": 0,
        "borderWidth": 1,
        "height": 14,
        "margin": 4,
        "width": 14,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 10,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`HighItems (with black item) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "black",
        "borderRadius": 0,
        "borderWidth": 1,
        "height": 14,
        "margin": 4,
        "width": 14,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 10,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "yellow",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1,
              "width": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
